Resources/Prototypes/Entities/Structures/Wallmounts/switch.yml

```diff
@@ -37,41 +37,41 @@
     snap:
     - Wallmount
   components:
-  - type: WallMount
-    arc: 360
-  - type: Clickable
-  - type: InteractionOutline
-  - type: Physics
-  - type: Sprite
-    sprite: Structures/Wallmounts/switch.rsi
-    state: dead
-  - type: UseDelay
-    delay: 0.5 # prevent light-toggling auto-clickers.
-  - type: SignalSwitch
-    onPort: Pressed
-    offPort: Pressed
-  - type: Rotatable
-  - type: Construction
-    graph: SignalButtonGraph
-    node: SignalButtonNode
-  - type: Fixtures
-  - type: SignalTransmitter
-    outputs:
-      Pressed: []
-  - type: Damageable
-    damageContainer: Inorganic
-    damageModifierSet: Metallic
-  - type: Destructible
-    thresholds:
-      - trigger:
-          !type:DamageTrigger
-          damage: 40
-        behaviors:
-          - !type:DoActsBehavior
-            acts: [ "Destruction" ]
-          - !type:PlaySoundBehavior
-            sound:
-              path: /Audio/Effects/metalbreak.ogg
+    - type: WallMount
+      arc: 360
+    - type: Clickable
+    - type: InteractionOutline
+    - type: Physics
+    - type: Sprite
+      sprite: Structures/Wallmounts/switch.rsi
+      state: dead
+    - type: UseDelay
+      delay: 0.5 # prevent light-toggling auto-clickers.
+    - type: SignalSwitch
+      onPort: Pressed
+      offPort: Pressed
+    - type: Rotatable
+    - type: Construction
+      graph: SignalButtonGraph
+      node: SignalButtonNode
+    - type: Fixtures
+    - type: SignalTransmitter
+      outputs:
+        Pressed: []
+    - type: Damageable
+      damageContainer: Inorganic
+      damageModifierSet: Metallic
+    - type: Destructible
+      thresholds:
+        - trigger:
+            !type:DamageTrigger
+            damage: 40
+          behaviors:
+            - !type:DoActsBehavior
+              acts: [ "Destruction" ]
+            - !type:PlaySoundBehavior
+              sound:
+                path: /Audio/Effects/metalbreak.ogg
```

Comment:
Same for these lines. Looks like only indentation was changed.

Reply:
see answer below